src/app/lib/config/services/marathi.js

```diff
@@ -178,19 +178,19 @@ export const service = {
         },
         {
           href: 'https://www.bbc.com/privacy/',
-          text: 'गोपनीयता धोरण',
+          text: 'गोपनीयतेचं धोरण',
         },
         {
           href: 'https://www.bbc.com/usingthebbc/cookies/',
           text: 'कुकीज',
         },
         {
           href: 'https://www.bbc.com/marathi/institutional-42227676',
-          text: 'बीबीसीला संपर्क करा',
+          text: 'बीबीसीशी संपर्क साधाा',
         },
       ],
       copyrightText:
-        'बीबीसी बाह्य इंटरनेट साइट्सच्या सामग्रीसाठी बीबीसी जबाबदार नाही',
+        'बीबीसी बाह्य इंटरनेट साइट्सच्या सामग्रीसाठी बीबीसी जबाबदार नाही.',
```

Comment:
The text in the issue is longer 
```
© 2019 BBC. बीबीसी बाह्य इंटरनेट साइट्सच्या सामग्रीसाठी बीबीसी जबाबदार नाही. बाह्य लिंक्सबद्दल आम्हाल काय वाटतं? इथे वाचा.
```
Is it just supposed to be this part?

Reply:
The other half is the `externalLink` text.

Reply:
They are combined in the Psammead Sitewide Link component:
```
    <StyledParagraph>
        {copyrightText}{' '}
        <Link text={externalLink.text} href={externalLink.href} inline />
      </StyledParagraph>
```